docs/source/examples/rshadows_tutorial.md

```diff
@@ -242,9 +248,7 @@ plt.xticks(
 plt.ylabel("Pauli fidelity")
 plt.legend()
 ```
-<!--     
-![png](../img/rshadows_compare_channels.png)
-     -->
```

Comment:
From the diff it seems that some images have been removed. Is this necessary?

Reply:
Ah so this is actually a markdown image that was commented out (surrounding by `<!--` and `-->`). I checked that `img/rshadows_compare_channels.png` does not exist, and the code above this part (and another) does generate the image. So just removing dead lines.

Reply:
Ah, I missed they were just comments. Thanks for removing them.